docs/GettingStartedDocs/Contributors/WindowsManualSGX1Prereqs.md

```diff
@@ -0,0 +1,17 @@
+# SGX1 mode: Prereqs
+
+## Intel SGX Platform Software for Windows (PSW) v2.2
+
+The PSW should be installed automatically on Windows 10 with the Fall Creators
+Update installed, or on a Windows Server 2016 image for an Azure Confidential
```

Comment:
- I'd suggest using versioning instead of the update names, so "Windows 10 version 1709 or newer".
- I would remove the reference to this being preinstalled on an WS2016 ACC VM image ... that's not the case as far as I know, and if it was, I don't think ACC would continue to support that custom template moving forwards. 
- This should converge with the version in WindowsSGX1GettingStarted.md (or just removed from there and consolidated here).

Reply:
Dropping SGX1 instructions as of now.